test/ComparedQueryable.Test/ComparedQueryableTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace ComparedQueryable.Test
+{
+    public class ComparedQueryableTest
+    {
+        [Fact]
+        public void NaturallySortsIEnumerableStrings()
+        {
+            // GIVEN I have a collection with a string property
+            var twoNorthStreet = new Address("2 N. Street");
+            var tenNorthStreet = new Address("10 N. Street");
+            var collection = new[] { twoNorthStreet, tenNorthStreet };

```

Comment:
I think it's worth having a test where the initial collection has tenNorthStreet so that we prove that we're actually sorting. Otherwise, the implementation could just return any results in the same order they were originally given.